pymc/sampling/mcmc.py

```diff
@@ -212,9 +212,79 @@ def all_continuous(vars):
         return True
 
 
+def sample_external_nuts(
+    sampler="nutpie",
+    draws: int = 1000,
+    initvals: Optional[Union[StartDict, Sequence[Optional[StartDict]]]] = None,
+    chains: Optional[int] = None,
+    cores: Optional[int] = None,
+    tune: int = 1000,
+    progressbar: bool = True,
+    model=None,
+    random_seed: RandomState = None,
```

Comment:
This is wrong, as nutpie only accepts a number, and from the main sample call your are passing a list of seeds already.

You can do `get_seeds_per_chain(random_seed, 1)[0]` or whatever it's called when passing the seed to nutpie.

Reply:
Thanks. Can't find `get_seeds_per_chain`.

Reply:
It's called `_get_seeds_per_chain` and it's already imported in this file